test/bufr_ncep_satwind_avhrr.yaml

```diff
@@ -0,0 +1,186 @@
+#(C) Copyright 2021 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# SAID, CLAT, CLON, YEAR, MNTH, DAYS, HOUR, MINU, SECO, SWCM, SAZA, OGCE, SCCF, SWQM
+# HAMD, PRLC, WDIR, WSPD
+# GNAP, PCCF
+
+observations:
+  - obs space:
+      name: bufr
+      #obsdatain: "./testinput/bufr_read_2_dim_blocks.bufr"
+      obsdatain: "/scratch1/NCEPDEV/da/Ashley.Stanfield/IODA/ioda-bundle/iodaconv/test/testinput/bufr_ncep_satwind_avhrr.bufr_d"
+
+      exports:
+        variables:
+          satelliteIdentifier:
+            query: "*/SAID"
+
+          latitude:
+            query: "*/CLAT"
+
+          longitude:
+            query: "*/CLON"
+
+          year:
+            query: "*/YEAR"
+
+          month:
+            query: "*/MNTH"
+
+          day:
+            query: "*/DAYS"
+
+          hour:
+            query: "*/HOUR"
+
+          minute:
+            query: "*/MINU"
+
+          second:
+            query: "*/SECO"
+
+          satelliteDerivedWindComputationMethod:
+            query: "*/SWCM"
+
+          satelliteZenithAngle:
+            query: "*/SAZA"
+
+          originatingCenter:
+            query: "*/OGCE"
+
+          satelliteChannelCentreFrequency:
+            query: "*/SCCF"
+
+          sdmeditSatWindQualMark:
+            query: "*/SWQM"
+
+          heightAssignmentMethod:
+            query: "*/HAMD[1]"
+
+          pressure:
+            query: "*/PRLC[1]"
+
+          windDirection:
+            query: "*/WDIR"
+
+          windSpeed:
+            query: "*/WSPD"
+
+          generatingApplication:
+            query: "*/GQCPRMS/GNAP[1]"
+
+          percentConfidence:
+            query: "*/GQCPRMS/PCCF[1]"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "/scratch1/NCEPDEV/da/Ashley.Stanfield/IODA/ioda-bundle/iodaconv/test/testoutput/bufr_ncep_satwind_avhrr.nc"
```

Comment:
@AshleyStanfield-NOAA The output should be stored in testrun directory, not under your account.